src/index.test.ts

```diff
@@ -181,30 +181,30 @@ describe('Video Reporter', () => {
     it('should add frame if no alert is displayed', async () => {
```

Comment:
Can we remove all `async` in tests where we don't use `await`?

Reply:
I think i got them all